atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -35,6 +39,9 @@
 WORKSPACE_KEY_BUILD_INFO = 'build_info'
 WORKSPACE_KEY_UPLOAD_DIR = 'koji_upload_dir'
 WORKSPACE_KEY_OVERRIDE_KWARGS = 'override_kwargs'
+FIND_CLUSTER_RETRY_DELAY = 1.0
```

Comment:
This is now redundant so let's remove it.

Reply:
This is different than FAILURE_RETRY_DELAY (or it will be). I just changed line 333 to use it instead of the failure delay.

Reply:
So the delay when attempting to list builds fails is different to the delay when attempting to create a build fails? Not sure I understand why.